db/schema.rb

```diff
@@ -67,6 +67,19 @@
     t.index ["item_id"], name: "index_reviews_on_item_id"
   end
 
+  create_table "users", force: :cascade do |t|
+    t.string "name"
+    t.string "address"
+    t.string "city"
```

Comment:
Looks great, glad we picked the one migration that hadnt been done.

Reply:
![](https://media.giphy.com/media/l2JebxHd2AkNMfUYM/giphy.gif)